dts/bindings/espi/realtek,rts5912-espi.yaml

```diff
@@ -17,3 +17,6 @@ properties:
 
   pinctrl-names:
     required: true
+
+  gpios:
```

Comment:
Name this `cs-gpios`.  You should also add some help text to describe this property.